finalfrontier/src/vocab.rs

```diff
@@ -2,7 +2,9 @@ use std::borrow::{Borrow, Cow};
 use std::collections::HashMap;
 use std::hash::Hash;
 
-use rust2vec::vocab::{SimpleVocab as R2VSimpleVocab, SubwordVocab as R2VSubwordVocab, VocabWrap};
+use finalfusion::vocab::{
+    SimpleVocab as R2VSimpleVocab, SubwordVocab as R2VSubwordVocab, VocabWrap,
```

Comment:
Should probably be `FiFuSubwordVocab` and `FiFuSimpleVocab`

Reply:
Indeed, fixed.